examples/applications/svm_gui.py

```diff
@@ -319,7 +319,13 @@ def get_parser():
 
 def main(argv):
     op = get_parser()
-    opts, args = op.parse_args(argv[1:])
+
```

Comment:
As @jnothman mentioned, I added it for the sake of consistency. Do you prefer I revert it?